ChangeLog

```diff
@@ -42,7 +42,7 @@
  * #1752 (Error while estimating the reduced log-likelihood when using a StationaryFunctionalCovarianceModel)
  * #1759 (Kriging model with StationaryFunctionalCovarianceModel might provide bad results)
  * #1768 (Bug in ExponentiallyDampedCosineModel & SphericalModel)
-
+ * #1771 (GridLayout hides the titles in the graph)
```

Comment:
you're deleting a line there, we want to avoid conflicts in that file

Reply:
hmm strange. I will fix it.